task-list-rails/app/views/tasks/show.html.erb

```diff
@@ -2,18 +2,68 @@
   <table>
     <thead>
         <tr>
-          <th colspan="2" id="title">Task details</th>
+          <td colspan="2" id="title">Task details</td>
+        </tr>
+        <tr>
+          <td colspan="2" id=edit_link><%= link_to "edit?", "/tasks/#{@task.id}/edit" %></td>
```

Comment:
Like the use of `link_to`! I'd suggest using a hash with routing information instead of the URL string, like: `{controller: "tasks", action: "edit", id: @task.id}`.